cookiecutter.json

```diff
@@ -14,7 +14,10 @@
     "version": "0.0.1-dev",
 
     "use_black_formatter": "n",
-    "use_spellcheck": "y",
+    "use_spellcheck": "n",
+
+    "use_tox": "n",
```

Comment:
No Tox by default?

Reply:
Nope, it is still an experimental feature (same as Makefile)